velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -339,10 +342,17 @@ class VectorFuzzer {
 
 /// Generates a random type, including maps, vectors, and arrays. maxDepth
 /// limits the maximum level of nesting for complex types. maxDepth <= 1 means
-/// no complex types are allowed.
-TypePtr randType(FuzzerGenerator& rng, int maxDepth = 5);
-
-/// Generates a random ROW type.
-RowTypePtr randRowType(FuzzerGenerator& rng, int maxDepth = 5);
+/// no complex types are allowed. allowIntervalType indicates whether
+/// INTERVAL_DAY_TIME type can be generated.
+/// (https://github.com/facebookincubator/velox/issues/6375)
+TypePtr
+randType(FuzzerGenerator& rng, int maxDepth = 5, bool allowIntervalType = true);
```

Comment:
looks good i wonder if instead of having allowIntervalType as function argument we shall have it as option in the fuzzer options since adding options as functions arguments does not scale

Reply:
same thing apply to int maxDepth = 5, but that we can do in adifferent diff